src/views/events/EventsDashboard.vue

```diff
@@ -118,8 +119,22 @@ function groupBy<T>(xs: T[], getKey: (element: T) => string) {
   }, {});
 }
 
+function findNextEventIndex() {
+  return actionsEvents?.value?.events.findIndex(
+    (event: Event) => new Date(event.end_time) > new Date(Date.now()),
+  );
+}
+
 const eventsWithFormattedDate = computed(() => {
-  const events = actionsEvents?.value?.events?.map((event: Event) => {
+  const urlParams = new URLSearchParams(window.location.search);
+  let earlier = 0;
+  if (urlParams.has("earlier")) {
+    earlier = Number(urlParams.get("earlier"));
+  }
```

Comment:
This is great, but there is an even more elegant solution that uses functions included in Vue and works without an if ;) Try this one:
```
const route = useRoute();
const earlier = Number(route.query.earlier ?? 0);
```